extension/lib/shopify/collection-repository/command/GetProductCountAdmin.js

```diff
@@ -0,0 +1,25 @@
+class ShopifyCollectionRepositoryCommandChildrenGetCount {
```

Comment:
The file should be moved into the "product" subfolder of folder "command" folder

Reply:
done